src/devtools/client/inspector/rules/components/DeclarationValue.tsx

```diff
@@ -10,12 +10,13 @@ interface DeclarationValueProps {
   colorSpanClassName: string;
   colorSwatchClassName: string;
   fontFamilySpanClassName: string;
+  priority?: string;
```

Comment:
Are there places where we'd like to render this without the priority? Chrome doesn't render it in its Computed tab